scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -956,7 +956,7 @@ class ScalaTestAntTask extends Task {
   // Class to hold data from <style> elements.
   //
   private class StyleElement {
-    private var name: String = null
+    private var name: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.